acct-user/vdr/metadata.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person" proxied="yes">
```

Comment:
You will be primary maintainer? (If so, that's perfectly okay)

Reply:
Yes... No.... I have never seen a definition of primary maintainer when more than one entry is there, or that the order of the entries should matter. But if it does, yes, I would do the job.